modules/doc/content/getting_started/installation/osx_other.md

```diff
@@ -0,0 +1,32 @@
+# Mac OS High Sierra or Older
+
+!include getting_started/minimum_requirements.md
+
+## Prerequisites
+
+The following, is required to be installed before you can begin using MOOSE.
+
+Due to the unfortunate difficulty Apple has placed on its users wishing to develop on platforms older than Mojave, it is strongly encouraged users upgrade to Mojave.
```

Comment:
I think the following should be at the top of the page.

!alert info title=Consider updating to Mojave
Due to the unfortunate difficulty Apple has placed on users wishing to develop on platforms older than Mojave, it is strongly encouraged users upgrade to Mojave.